lbrynet/lbrynet_daemon/LBRYDaemonServer.py

```diff
@@ -29,6 +33,184 @@
 log.addHandler(handler)
 log.setLevel(logging.INFO)
 
+class LBRYDaemonRequest(server.Request):
```

Comment:
This class is large enough that it should live in its own module.

Huge modules are harder to read.